test/Ocelot.UnitTests/DependencyInjection/ConfigurationBuilderExtensionsTests.cs

```diff
@@ -36,7 +36,7 @@ public ConfigurationBuilderExtensionsTests()
             _hostingEnvironment = new Mock<IWebHostEnvironment>();
 
             // Clean up config files before each test
-            var subConfigFiles = new DirectoryInfo(".").GetFiles("ocelot.*.json");
+            var subConfigFiles = new DirectoryInfo(".").GetFiles("ocelot*.json");

```

Comment:
Why did you remove the dot (`.` sign) from file template?
How will it reflect to current docs? See: [Merging configuration files](../../blob/develop/docs/features/configuration.rst#merging-configuration-files)

Reply:
not an issue!
The code removes all files of the old/previous test.